src/modules/commands/discord/outfitstats.ts

```diff
@@ -0,0 +1,189 @@
+import discord from 'discord.js'
+import { OutfitAliasNotFoundError } from '@app/errors'
+import { divide } from '@app/utils/math'
+import { Command } from '@commands/CommandHandler'
+import { getEmoji } from '@discord/utils'
+import { censusApi } from '@planetside/CensusApi'
+import { validateArgumentRange } from '../validators'
+
+export default new Command<discord.Message>({
+  keyword: 'outfitstats',
+  description: 'show PS2 outfit stats',
+  help: "Usage: `{prefix}outfitstats <alias>` - shows an outfit's players average stats\n{prefix}outfitstats <alias> active` - shows an outfit's active players average stats",
```

Comment:
There should be a `\n` after 'Usage:', and the last row is missing backticks around the command.